install/kubernetes/Makefile.digests

```diff
@@ -1,13 +1,13 @@
-# File generated by .github/workflows/images-legacy-releases.yaml; DO NOT EDIT.
+# File generated by .github/workflows/build-images-releases.yaml; DO NOT EDIT.
 # Copyright Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-CILIUM_DIGEST := ""
-CLUSTERMESH_APISERVER_DIGEST := ""
-DOCKER_PLUGIN_DIGEST := ""
-HUBBLE_RELAY_DIGEST := ""
-OPERATOR_AWS_DIGEST := ""
-OPERATOR_AZURE_DIGEST := ""
-OPERATOR_ALIBABACLOUD_DIGEST := ""
-OPERATOR_GENERIC_DIGEST := ""
-OPERATOR_DIGEST := ""
+export CILIUM_DIGEST := ""
```

Comment:
The alternative to all this exporting is to just blanket export all variables to subshells via `.EXPORT_ALL_VARIABLES` as a [Special Target](https://www.gnu.org/software/make/manual/html_node/Special-Targets.html), but I wasn't sure if that might cause other issues since I don't know if that's scoped only to this Makefile or whether it could end up exporting other vars.